packages/auth-construct/src/construct.ts

```diff
@@ -504,13 +531,23 @@ export class AmplifyAuth
         ...customAttributes,
       },
       email: props.senders
-        ? cognito.UserPoolEmail.withSES({
-            fromEmail: props.senders.email.fromEmail,
-            fromName: props.senders.email.fromName,
-            replyTo: props.senders.email.replyTo,
-            sesRegion: Stack.of(this).region,
-          })
+        ? (() => {
+            if ('fromEmail' in props.senders.email) {
+              return cognito.UserPoolEmail.withSES({
+                fromEmail: props.senders.email.fromEmail,
+                fromName: props.senders.email.fromName,
+                replyTo: props.senders.email.replyTo,
+                sesRegion: Stack.of(this).region,
+              });
+            }
+            return undefined;
+          })()
         : undefined,
+      lambdaTriggers: {
```

Comment:
we should have a test to make sure that setting lambdaTriggers to {} does not remove any triggers that get added elsewhere